src/main/webapp/app/course/course-scores/course-scores.component.ts

```diff
@@ -248,26 +253,31 @@ export class CourseScoresComponent implements OnInit, OnDestroy {
         const courseScoresObservable = this.participantScoresService.findCourseScores(courseId);
         // find grading scale if it exists for course
         const gradingScaleObservable = this.gradingSystemService.findGradingScaleForCourse(courseId).pipe(catchError(() => of(new HttpResponse<GradingScale>())));
-        forkJoin([findParticipationsObservable, courseScoresObservable, gradingScaleObservable]).subscribe(([participationsOfCourse, courseScoresResult, gradingScaleResponse]) => {
-            this.allParticipationsOfCourse = participationsOfCourse;
+        const plagiarismCasesObservable = this.plagiarismCasesService.getCoursePlagiarismCasesForInstructor(courseId);
+        forkJoin([findParticipationsObservable, courseScoresObservable, gradingScaleObservable, plagiarismCasesObservable]).subscribe(
+            ([participationsOfCourse, courseScoresResult, gradingScaleResponse, plagiarismCases]) => {
+                this.allParticipationsOfCourse = participationsOfCourse;
 
-            this.calculateExerciseLevelStatistics();
-            this.exerciseTypesWithExercises = this.filterExercisesTypesWithExercises();
+                this.calculateExerciseLevelStatistics();
+                this.exerciseTypesWithExercises = this.filterExercisesTypesWithExercises();
 
-            this.calculateStudentLevelStatistics();
+                this.calculateStudentLevelStatistics();
 
-            // if grading scale exists set properties
-            if (gradingScaleResponse.body) {
-                this.calculateGradingScaleInformation(gradingScaleResponse.body);
-            }
+                // if grading scale exists set properties
+                if (gradingScaleResponse.body) {
+                    this.calculateGradingScaleInformation(gradingScaleResponse.body, plagiarismCases.body || undefined);
```

Comment:
Why is `|| undefined` needed?

Reply:
Removing it gives the following error for the target method with signature `calculateGradingScaleInformation(gradingScale: GradingScale, plagiarismCases?: PlagiarismCase[])`:

```
TS2345: Argument of type 'PlagiarismCase[] | null' is not assignable to parameter of type 'PlagiarismCase[] | undefined'.   Type 'null' is not assignable to type 'PlagiarismCase[] | undefined'.
```

Reply:
Can you use `?? undefined` here? Js considers an empty array as falsy, but I think this should get passed to the method.
![grafik](https://user-images.githubusercontent.com/26540346/208455053-1ae1c1a4-e898-42de-9406-04d4aa155660.png)

Reply:
Done 👍